docs/recipes/relion.md

```diff
@@ -0,0 +1,154 @@
+---
+tags:
+ - Satori
+ - MPI
+ - RELION
+ - Howto Recipes
+---
+
+# Installing and Using RELION
+
+RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a software package that employs an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination. 
+
+## RELION on Satori 
+
+This recipe is for building and using RELION on x86 nodes on Satori. It is different from working on IBM power9 nodes on Satori. 
+
+> Note: The x86 nodes are available to some labs only. 
```

Comment:
Did you want to use the [admonition syntax](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) for the Notes, as you did in the MPI pages? This one:

!!! Note
    The x86 nodes are available to some labs only.